config/routes.rb

```diff
@@ -1,3 +1,19 @@
 Rails.application.routes.draw do
+  root 'events#index'
+  get 'signup' => 'users#new', as: 'signup'
+  get 'login' => 'sessions#new', as: 'login'
+  get 'logout' => 'sessions#destroy', as: 'logout'
+  resources :events do
+    member do
+      get 'attend'
+      get 'cancel_attend'
+    end
+  end
```

Comment:
- [ ] Lets also ensure that  routes are only for used actions, i.e. `resources :events, only: [:new, :create, :show, :index]`